### PR TITLE
[IMP] barcode: split docs + hone in on language

### PR DESCRIPTION
**Pull request by Felicious**
### Summary

Improve wording about GS1 barcode concepts.
- Received [feedback](https://www.odoo.com/web#id=3292038&cids=3&model=project.task&view_type=form) from CRU about barcode conceptual accuracy and changed the wording to reflect it
- Created scannable 2D matrix barcodes (they look like QR codes) for readers to try scanning!
- Received feedback from hojo to split the docs into two so it's easier for readers to search for GS1 nomenclature (setup) vs. using GS1 barcodes in their warehouse workflow.
- Received [feedback](https://www.odoo.com/web#id=3474304&cids=3&model=project.task&view_type=form) from ZST about honing in on language surrounding creating GS1 barcodes. Need to make it very clear that Odoo **doesn't** generate GS1 barcodes for users! We just take in their inputs. Can reviewers make sure that my doc makes this very clear? (:

Thanks :smile_cat:

### CODE REVIEW

File: content/applications/inventory_and_mrp/inventory/barcode/operations/gs1_nomenclature.rst [hunk 1]
```diff
@@ -2,13 +2,21 @@
...
+.. |AI| replace:: :abbr:`A.I. (Application Identifier)`
+.. |GTIN| replace:: :abbr:`GTIN (Global Trade Item Number)`
+
+`GS1 nomenclature <https://www.gs1us.org/>`_ consolidates various product and supply chain data into
+a single barcode. Odoo takes in `unique Global Trade Item Numbers
+<https://www.gs1.org/standards/get-barcodes>`_ (GTIN), purchased by businesses, to enable global
+shipping, sales, and eCommerce product listing.
+
+Configure GS1 nomenclature to scan barcodes of sealed boxes and identify essential product
+information, such as |GTIN|, lot number, quantity information, and more.
```
Comment (Felicious) [inline content/applications/inventory_and_mrp/inventory/barcode/operations/gs1_nomenclature.rst:16]: @StraubCreative Do you think this makes it clear enough that GTINs must be unique and purchased by GS1?

File: content/applications/inventory_and_mrp/inventory/barcode/operations/gs1_nomenclature.rst [hunk 2]
```diff
@@ -25,214 +33,53 @@ barcode nomenclature options.
...
+=======================
 
-.. example::
-   To create a barcode for the product, `Fuji Apple`, enter the 14-digit GTIN `12345678901231` in
-   the :guilabel:`Barcode` field on the product form.
+For product identification using GS1 barcodes in Odoo, businesses obtain a `unique GTIN
+<https://www.gs1.org/standards/get-barcodes>`_ as an internationally distinct product identifier
+purchased from GS1. This |GTIN| is combined with specific product details following GS1's designated
+*barcode pattern*. The barcode pattern's arrangement of numbers and letters must adhere to GS1
+conventions for accurate interpretation by global systems along the supply chain.
```
Comment (Felicious) [inline content/applications/inventory_and_mrp/inventory/barcode/operations/gs1_nomenclature.rst:60]: @StraubCreative Do you think this makes it clear enough about GTINs that average Joe eCommerce dumdums wouldn't get confused?

File: content/applications/inventory_and_mrp/inventory/barcode/operations/gs1_usage.rst [hunk 3]
```diff
@@ -0,0 +1,240 @@
...
+globally recognized across the supply chain. In Odoo, GS1 barcodes streamline various warehouse
+processes, such as receiving packages, by automating data entry and reducing processing times.
+
+The following sections contain examples of how Odoo uses GS1 barcodes provided by the business to
+identify common warehouse items and automate certain warehouse workflows.
+
+.. important::
+   Odoo **does not** create GS1 barcodes. Businesses must provide a unique Global Trade Item Number
+   (GTIN) from GS1. Then, they can combine their existing GS1 barcodes with product and supply chain
+   information (also provided by GS1) to create barcodes in Odoo.
```
Comment (Felicious) [inline content/applications/inventory_and_mrp/inventory/barcode/operations/gs1_usage.rst:20]: @StraubCreative Do you think this makes it clear that GTINs are unique and purchased from GS1?

File: content/applications/inventory_and_mrp/inventory/barcode/operations/gs1_usage.rst [hunk 4]
```diff
@@ -0,0 +1,240 @@
...
+        - |AI|
+          (quantity)
+        - GS1 Barcode (quantity)
+        - |AI| (lot)
+        - GS1 Barcode (lot #)
+        - Full GS1 barcode
+      * - 50 Fuji apples in Lot0002
+        - .. image:: gs1_usage/fuji-apples-barcode.png
+             :align: center
+             :alt: 2D matrix of GS1 barcode of 50 fuji apples with an assigned lot number.
```
Comment (Felicious) [inline content/applications/inventory_and_mrp/inventory/barcode/operations/gs1_usage.rst:139]: @samueljlieber Do you have any suggestions about fixing the image width/heights or scaling the barcode? Even at full size, the barcode image is a little small and I'm wondering if it'll be an issue when the window needs to be minimized/resized.

File: content/applications/inventory_and_mrp/inventory/barcode/operations/gs1_usage.rst [hunk 5]
```diff
@@ -0,0 +1,240 @@
...
+        - 01
+        - 20611628936004
+        - 30
+        - 00000050
+        - 10
+        - LOT002
+        - 01206116289360043
+          000000050
+          10LOT0002
```
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/inventory/barcode/operations/gs1_usage.rst:148]:
```suggestion
   .. list-table::
      :widths: 50 50
      :header-rows: 1
      :stub-columns: 1

      * - 50 Fuji apples in Lot0002
        -
      * - 2D Matrix
        - .. image:: gs1_usage/fuji-apples-barcode.png
             :alt: 2D matrix of GS1 barcode of 50 fuji apples with an assigned lot number.
      * - |AI| (product)
        - 01
      * - GS1 Barcode (product)
        - 20611628936004
      * - |AI| (quantity)
        - 30
      * - GS1 Barcode (quantity)
        - 00000050
      * - |AI| (lot)
        - 10
      * - GS1 Barcode (lot #)
        - LOT0002
      * - Full GS1 barcode
        - 01206116289360043 000000050 10LOT0002
```

File: content/applications/inventory_and_mrp/inventory/barcode/operations/gs1_usage.rst (hunk 4 above)
Comment (samueljlieber) [reply content/applications/inventory_and_mrp/inventory/barcode/operations/gs1_usage.rst:139]: I agree the QR code and table headers do appear to be scrunched in this layout. I'd suggest flipping the table axis so there are two columns, giving more room to breathe and accommodate different screen sizes. I added RST here: https://github.com/odoo/documentation/pull/5592/files/73cd08835811c5e774d4ac0e58fad9e38ac48cb2#r1305710601

Screenshot 2023-08-25 at 10 06 49 AM

File: content/applications/inventory_and_mrp/inventory/barcode/operations/gs1_usage.rst [hunk 6]
```diff
@@ -0,0 +1,233 @@
...
+
+Setting up GS1 barcodes in Odoo enables businesses to configure their products and make them
+globally recognized across the supply chain. In Odoo, GS1 barcodes streamline various warehouse
+processes, such as receiving packages, by automating data entry and reducing processing times.
+
+The following sections contain examples of how Odoo uses GS1 barcodes provided by the business to
+identify common warehouse items and automate certain warehouse workflows.
+
+.. important::
+   Odoo **does not** create GS1 barcodes. Businesses must provide a unique Global Trade Item Number
```
Comment (brse-odoo) [inline content/applications/inventory_and_mrp/inventory/barcode/operations/gs1_usage.rst:18]:
```suggestion
   Odoo **does not** create GS1 barcodes. Businesses must purchase a unique Global Trade Item Number
```

File: content/applications/inventory_and_mrp/inventory/barcode/operations/gs1_usage.rst [hunk 7]
```diff
@@ -0,0 +1,233 @@
...
+
+Scan barcode on receipt
+-----------------------
+
+To ensure accurate lot interpretation in Odoo on product barcodes scanned during a receipt
+operation, navigate to the :menuselection:`Barcode` app to manage the :ref:`receipt picking process
+<barcode/operations/scan-received-products>`.
+
+From the :guilabel:`Barcode Scanning` dashboard, click the :guilabel:`Operations` button, then the
+:guilabel:`Receipts` button to view the list of vendor receptions to process. Receipts generated
```
Comment (brse-odoo) [inline content/applications/inventory_and_mrp/inventory/barcode/operations/gs1_usage.rst:105]:
```suggestion
:guilabel:`Receipts` button to view the list of vendor receipts to process. Receipts generated
```
Which sounds better?

File: content/applications/inventory_and_mrp/inventory/barcode/operations/gs1_usage.rst [hunk 8]
```diff
@@ -0,0 +1,233 @@
...
+   :guilabel:`Units` processed will be displayed and the :guilabel:`Validate` button turns green.
+   Click the :guilabel:`Validate` button to complete the reception.
+
+   .. image:: gs1_usage/receive-50-apples.png
+      :align: center
+      :alt: Scan the barcode for a product on the reception picking page in the *Barcode* app.
+
+.. _barcode/operations/quantity-ex:
+
+Product + non-unit quantity
```
Comment (brse-odoo) [inline content/applications/inventory_and_mrp/inventory/barcode/operations/gs1_usage.rst:156]: Same note for the similar heading above. But if you want to stick with this style heading then feel free, it's up to you

**Review by brse-odoo — COMMENTED** [review]
@Felicious I've just finished my review of this doc, great work! I left suggestions for grammar/clarity, so feel free to ask me about any of them, as always. Thanks!

File: content/applications/inventory_and_mrp/inventory/barcode/operations/gs1_usage.rst [hunk 9]
```diff
@@ -0,0 +1,233 @@
...
+   (GTIN) from GS1. Then, they can combine their existing GS1 barcodes with product and supply chain
+   information (also provided by GS1) to create barcodes in Odoo.
+
+.. seealso::
+   - `Purchase GTINs <https://www.gs1.org/standards/get-barcodes>`_
+   - :ref:`GS1 nomenclature <barcode/operations/gs1>`
+
+.. _barcode/operations/gs1-lots:
+
+Product + quantity + lot
```
Comment (Felicious) [inline content/applications/inventory_and_mrp/inventory/barcode/operations/gs1_usage.rst:28]:
```suggestion
Configure barcodes for product, quantity, and lots
```

File: content/applications/inventory_and_mrp/inventory/barcode/operations/gs1_usage.rst (hunk 8 above)
Comment (Felicious) [inline content/applications/inventory_and_mrp/inventory/barcode/operations/gs1_usage.rst:156]:
```suggestion
Configure barcode for product and non-unit quantity
```

File: content/applications/inventory_and_mrp/inventory/barcode/operations/gs1_nomenclature.rst [hunk 10]
```diff
@@ -25,214 +33,53 @@ barcode nomenclature options.
...
-
-.. example::
-   After placing a :abbr:`PO (Purchase Order)` for 50 apples, navigate to the associated receipt.
-   Scan the product barcode, and Odoo will prompt for the lot number.
-
-   .. image:: gs1_nomenclature/receive-50-apples.png
-      :align: center
-      :alt: Scan the barcode for a product on the reception picking page in the *Barcode* app.
+   When using a flexible length barcode pattern, and the barcode is not placed at the end of the GS1
+   barcode, use the FNC1 separator (`\\x1D`) to end the barcode without needing to reach the maximum
```
[the remark this answers is not in the document]
Comment (Felicious) [reply content/applications/inventory_and_mrp/inventory/barcode/operations/gs1_nomenclature.rst:72]: Added an example to illustrate this :D

File: content/applications/inventory_and_mrp/inventory/barcode/operations/gs1_usage.rst [hunk 11]
```diff
@@ -0,0 +1,233 @@
...
+GS1 barcode usage
+=================
+
+.. _barcode/operations/gs1_usage:
+
+.. |AI| replace:: :abbr:`A.I. (Application Identifier)`
+.. |GTIN| replace:: :abbr:`GTIN (Global Trade Item Number)`
+
+Setting up GS1 barcodes in Odoo enables businesses to configure their products and make them
+globally recognized across the supply chain. In Odoo, GS1 barcodes streamline various warehouse
```
[the remark this answers is not in the document]
Comment (Felicious) [reply content/applications/inventory_and_mrp/inventory/barcode/operations/gs1_usage.rst:11]: I rewrote the intro! What do you think?

> GS1 barcodes are a standardized language for barcode scanners helping businesses with international
supply chain operations. Use Odoo *Barcode* to interpret and print GS1 barcodes, making warehouse
operations, like receiving and picking, more efficient.

**Comment by Felicious** [comment]
Hi @brse-odoo ! This PR is ready for your second review (: I rewrote the intro of the barcode usage doc, as per your input. Let me know if it's an improvement towards the direction you were pointing in!

Thank you :D

File: content/applications/inventory_and_mrp/inventory/barcode/operations/gs1_usage.rst [hunk 12]
```diff
@@ -0,0 +1,234 @@
+=================
+GS1 barcode usage
+=================
+
+.. _barcode/operations/gs1_usage:
+
+.. |AI| replace:: :abbr:`A.I. (Application Identifier)`
+.. |GTIN| replace:: :abbr:`GTIN (Global Trade Item Number)`
+
+GS1 barcodes are a standardized language for barcode scanners helping businesses with international
```
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/barcode/operations/gs1_usage.rst:10]: - singular/plural mismatch
- *language* is the wrong word here I think?
- this first sentence is a bit clunky 🥴

File: content/applications/inventory_and_mrp/inventory/barcode/operations/gs1_usage.rst [hunk 13]
```diff
@@ -0,0 +1,234 @@
...
+.. seealso::
+   - `Purchase GTINs <https://www.gs1.org/standards/get-barcodes>`_
+   - :ref:`GS1 nomenclature <barcode/operations/gs1>`
+
+.. _barcode/operations/gs1-lots:
+
+Configure barcodes for product, quantity, and lots
+==================================================
+
+To build a GS1 barcode for a box that contains a product, number of units in it, and the lot number,
```
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/barcode/operations/gs1_usage.rst:31]: The way this is phrased implies that all three variables are necessary in a GS1 barcode. I don't think they are?

What about information like batch number, expiration dates, etc., which can also be included in GS1's?

I'd rephrase this intro sentence so it's not forcing a specific scenario:

```suggestion
To build a GS1 barcode for a shipment, use the following barcode patterns and Application Identifiers (A.I.) in the table below:
```

For a future PR, especially since the apple is a good example: I wonder if you can include expiration dates too here?

File: content/applications/inventory_and_mrp/inventory/barcode/operations/gs1_usage.rst [hunk 14]
```diff
@@ -0,0 +1,234 @@
...
+
+.. _barcode/operations/lot-setup:
+
+Configuration
+-------------
+
+To track products using lots, first enable the :ref:`Lots and Serial Numbers
+<inventory/management/track_products_by_lots>` feature. To do so, navigate to
+:menuselection:`Inventory app --> Configuration --> Settings`. Next, under the
+:guilabel:`Traceability` heading, check the box for :guilabel:`Lots & Serial Numbers`.
```
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/barcode/operations/gs1_usage.rst:54]: Condense
```suggestion
First, :ref:`enable product tracking using lots <inventory/management/track_products_by_lots>` by
navigating to :menuselection:`Inventory app --> Configuration --> Settings`, and checking the box
for :guilabel:`Lots & Serial Numbers` under the :guilabel:`Traceability` heading.
```

File: content/applications/inventory_and_mrp/inventory/barcode/operations/gs1_usage.rst [hunk 15]
```diff
@@ -0,0 +1,234 @@
...
+   :guilabel:`Validate` to complete the validation.
+
+   .. image:: gs1_usage/scan-barcode-peaches.png
+      :align: center
+      :alt: Scan barcode screen for a reception operation in the Barcode app.
+
+Verify product moves
+====================
+
+For additional verification purposes, the quantities of received products are also recorded on the
```
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/barcode/operations/gs1_usage.rst:222]:
```suggestion
For additional verification, the quantities of received products are also recorded on the
```

**Review by StraubCreative — CHANGES_REQUESTED** [review]
Hi @Felicious 
A couple necessary + optional change requests below.
I like the QR code examples, very clever 😉 

Think this PR should be good to ship on next commit.
Thanks!

File: content/applications/inventory_and_mrp/inventory/barcode/operations/gs1_nomenclature.rst [hunk 16]
```diff
@@ -25,214 +38,56 @@ barcode nomenclature options.
...
+along with the corresponding :guilabel:`Barcode Pattern`.
 
 .. tip::
    After setting GS1 as the barcode nomenclature, :menuselection:`Barcode Nomenclatures` can also be
-   accessed by first enabling :ref:`developer mode <developer-mode>`. Navigate to
-   :menuselection:`Inventory app --> Configuration --> Barcode Nomenclatures` and finally, select
-   :guilabel:`Default GS1 Nomenclature`.
+   accessed by first enabling :ref:`developer mode <developer-mode>`. Navigate to the
+   :menuselection:`Inventory app --> Configuration --> Barcode Nomenclatures` menu and finally,
+   select :guilabel:`Default GS1 Nomenclature`.
```
[the remark this answers is not in the document]
Comment (Felicious) [reply content/applications/inventory_and_mrp/inventory/barcode/operations/gs1_nomenclature.rst:54]: "hidden menu" ! what exciting language, i love it :D

File: content/applications/inventory_and_mrp/inventory/barcode/operations/gs1_usage.rst (hunk 12 above)
Comment (Felicious) [reply content/applications/inventory_and_mrp/inventory/barcode/operations/gs1_usage.rst:10]: what do u think about my intro rewrite?
> 
GS1 barcodes provide a standardized format that barcode scanners can interpret. They encode
information in a :ref:`specific structure recognized globally <barcode/operations/gs1>`, allowing
scanners to understand and process supply chain data consistently.

> Odoo *Barcode* interprets and prints GS1 barcodes, automating product identification and tracking
in warehouse operations such as receiving, picking, and shipping.

Comment (StraubCreative) [reply content/applications/inventory_and_mrp/inventory/barcode/operations/gs1_usage.rst:10]: good rewrite
one revision: 
`pecific structure recognized globally` --> `a specific, globally recognized structure`

File: content/applications/inventory_and_mrp/inventory/barcode/operations/gs1_nomenclature.rst [hunk 17]
```diff
@@ -289,6 +144,14 @@ not working as expected:
    the barcode nomenclature for Odoo to read the barcode. :ref:`This section
    <barcode/operations/create-new-rules>` details how to add new rules in the barcode nomenclature.
 
+#. Test barcodes containing multiple encoded fields, piece by piece to figure out which field is
```
[the remark this answers is not in the document]
Comment (Felicious) [reply content/applications/inventory_and_mrp/inventory/barcode/operations/gs1_nomenclature.rst:147]: Moved this paragraph to the Troubleshooting section, and rewrote the intro of the Create rules section.

> GS1 rules are a specific format of information contained in the barcode, beginning with an |AI| and
containing a defined length of characters. Scanning GS1 barcodes from the :ref:`default GS1 list
<barcode/operations/default-gs1-nomenclature-list>` auto-fills corresponding data in the Odoo
database.

> Adding GS1 barcode rules in Odoo ensures accurate interpretation of unique, non-standard GS1
formats.

> To do so,

**Review by StraubCreative — APPROVED** [review]
Looks good @Felicious, nice work 🚀 

@robodoo r+

**Comment by StraubCreative** [comment]
@robodoo retry
